deploy/examples/create-external-cluster-resources.py

```diff
@@ -1222,14 +1143,40 @@ def gen_json_out(self):
                     "userKey": self.out_map['ROOK_EXTERNAL_DASHBOARD_LINK']
                 }
             })
+        if self.out_map['RBD_METADATA_EC_POOL_NAME']:
+            json_out.append({
+                "name": "ceph-rbd",
+                "kind": "StorageClass",
+                "data": {
+                    "dataPool": self.out_map['RBD_POOL_NAME'],
+                    "pool": self.out_map['RBD_METADATA_EC_POOL_NAME'],
```

Comment:
This need to be inter changed. EC pool need to be mentioned as dataPool

Reply:
Yes right I also doubt when seeing it, but this is how it's updated by @subhamkrai by this commit https://github.com/rook/rook/commit/22865815cf98e8af4357591e4c609e2c87c35d9e, so any specific reason for keeping it like that.

Reply:
This will not work with cephcsi.

Reply:
Yes right